gyp/pylib/gyp/generator/cmake.py

```diff
@@ -239,6 +241,8 @@ def StringToCMakeTargetName(a):
   Invalid for make: ':'
   Invalid for unknown reasons but cause failures: '.'
   """
+  if PY3:
+      return a.translate(str.maketrans(' /():."', '_______'))
   return a.translate(string.maketrans(' /():."', '_______'))
```

Comment:
Nice!!  The more canonical way to do this is to remove line 40 and then:
```
try:
    return a.translate(str.maketrans(' /():."', '_______'))
except AttributeError:
    return a.translate(string.maketrans(' /():."', '_______'))
```
https://docs.python.org/3/howto/pyporting.html#use-feature-detection-instead-of-version-detection

Reply:
My thought is that we can to remove python2 related code with searching `PY3` :)